back/app/uploaders/event_file_uploader.rb

```diff
@@ -2,7 +2,8 @@
 
 class EventFileUploader < BaseFileUploader
   def extension_allowlist
-    %w[avi csv doc docx key mkv mp3 mp4 numbers odp ods odt pages pdf ppt pptx txt xls xlsx]
+    %w[avi csv doc docx key mkv mp3 mp4 numbers odp ods odt pages pdf ppt pptx txt xls xlsx] +
+      BaseImageUploader::ALLOWED_TYPES
```

Comment:
It seems like the only place we diverge from this is in `IdeaImportFileUploader` [here](https://github.com/CitizenLabDotCo/citizenlab/blob/c53a18d992ba6f1cd190042dbd19471f05babb89/back/engines/commercial/bulk_import_ideas/app/uploaders/bulk_import_ideas/idea_import_file_uploader.rb#L6).

Should we remove these duplicate overrides of  `extension_allowlist` method, which are actually identical to the method in `FileUploader` they override?

Reply:
Yes, there is probably some room for simplification. I would definitely the methods if they are identical to that the parent's. I would maybe keep the different Uploaders for now unless you're confident that they are all identical, now and for the foreseeable future.

Reply:
> I would definitely the methods
What would you definitely do with the methods? 😄

Reply:
remove them 😅